site/docs/components/slider/usage.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Using the component
+
+### When to use
+
+- When selecting a single value, range of numeric values or relative scale.
+- When the exact value does not matter (depending on the size of the predefined range), but an approximate value is good enough.
+- When changes are applied for immediate feedback, e.g., changing the volume on an audio or video player.
+
+### When not to use
+
+- When it’s important to select an exact numeric value that falls within a significantly large range, for example, 1-1000. Instead, use Input.
+- When displaying progress through a series of separate steps, such as a wizard or multistep form. Use Stepped tracker instead.
+
+### Content
+
+Display labels as numbers so they remain as short as possible and don't overlap with each other or with the Slider. If labels are overlapping, adjust the Slider width and/or number of visible increments.
```

Comment:
"Labels" doesn't seem to be a concept in slider

Reply:
Labels are auto-generated by the component, so this isn't even a thing anymore. Have removed 👍